src/js/components/stock-movement-wizard/outboundImport/subsections/OutboundImportShipmentDetails.jsx

```diff
@@ -0,0 +1,153 @@
+                  defaultMessage: 'Origin',
+                }}
+                hasErrors={Boolean(errors.origin?.message)}
+                errorMessage={errors.origin?.message}
+              />
+            )}
+          />
+        </div>
+        <div className="col-lg-3 col-md-6 px-2 pt-2">
+          <Controller
+            name="destination"
+            control={control}
+            render={({ field }) => (
+              <SelectField
+                {...field}
+                disabled
+                placeholder="Select"
+                title={{
+                  id: 'react.stockMovement.destination.label',
+                  defaultMessage: 'Destination',
+                }}
+                hasErrors={Boolean(errors.destination?.message)}
+                errorMessage={errors.destination?.message}
+              />
+            )}
+          />
+        </div>
+        <div className="col-lg-3 col-md-6 px-2 pt-2">
+          <Controller
+            name="shipDate"
+            control={control}
+            render={({ field }) => (
+              <DateField
+                {...field}
+                disabled
+                title={{
+                  id: 'react.stockMovement.shipDate.label',
+                  defaultMessage: 'Ship date',
+                }}
+                placeholder={{
+                  id: 'react.default.dateInput.placeholder.label',
+                  default: 'Select a date',
+                }}
+                errorMessage={errors.dateShipped?.message}
+              />
+            )}
+          />
+        </div>
+        <div className="col-lg-3 col-md-6 px-2 pt-2">
+          <Controller
+            name="shipmentType"
+            control={control}
+            render={({ field }) => (
+              <SelectField
+                {...field}
+                disabled
+                placeholder="Select"
+                title={{
+                  id: 'react.stockMovement.shipmentType.label',
+                  defaultMessage: 'Shipment type',
+                }}
+                hasErrors={Boolean(errors.shipmentType?.message)}
+                errorMessage={errors.shipmentType?.message}
+              />
+            )}
+          />
+        </div>
+        <div className="col-lg-3 col-md-6 px-2 pt-2">
+          <Controller
+            name="trackingNumber"
+            control={control}
+            render={({ field }) => (
+              <TextInput
+                {...field}
+                disabled
+                title={{
+                  id: 'react.stockMovement.trackingNumber.label',
+                  defaultMessage: 'Tracking Number',
+                }}
+                errorMessage={errors.trackingNumber?.message}
+              />
+            )}
+          />
+        </div>
+        <div className="col-lg-3 px-2 pt-2 d-lg-block d-md-none" />
+        <div className="col-lg-3 px-2 pt-2 d-lg-block d-md-none" />
+        <div className="col-lg-3 col-md-6 px-2 pt-2">
```

Comment:
what are those two empty divs responsible for?

Reply:
Let me share a screenshot for how the confirm page looks like first
![image](https://github.com/openboxes/openboxes/assets/35233347/d559321c-b667-447d-8bd8-fa704a7bd84e)
So there are supposed to be a space between two input, and these two empty divs just act as a placeholders empty spaces for the grid.

To answer your potential question "Why did you not use **space-between**?" it's because when using space-between it will not have the same effect and will result in these two fields be of different size than the ones above since they are going to be pushed to the smallest element size which is different to the auto-fit size of the grid elements.

Reply:
sounds good